src/main/java/org/fcrepo/auth/webac/WebACRolesProvider.java

```diff
@@ -465,15 +431,15 @@ private static WebACAuthorization createAuthorizationFromMap(final Map<String, L
         final Map<String, List<String>> aclTriples = new HashMap<>();
         final List<WebACAuthorization> authorizations = new ArrayList<>();
 
-        getDefaultAcl().listStatements().forEachRemaining(x -> {
-            if (isAclPredicate.test(x.getPredicate())) {
-                final Triple t = x.asTriple();
-                aclTriples.putIfAbsent(t.getPredicate().getURI(), new ArrayList<>());
-                if (t.getObject().isURI()) {
-                    aclTriples.get(t.getPredicate().getURI()).add(t.getObject().getURI());
-                } else if (t.getObject().isLiteral()) {
-                    aclTriples.get(t.getPredicate().getURI()).add(
-                        t.getObject().getLiteralValue().toString());
+        getDefaultAcl().listStatements().forEachRemaining(statement -> {
```

Comment:
```
listStatements().mapWith(Statement::asTriple).filterKeep(hasAclPredicate).
    forEachRemaining(…
```
reads better to me, or, if that doesn't work, we really should think about putting a utility class in place for pre-Jena 3 `Filter`s from Java 8 lambdas and method refs, because it keeps coming up. Or we should do the work for Jena 3, he said without volunteering. (At least `mapWith`can be used here.)